civis/ml/_model.py

```diff
@@ -907,7 +908,8 @@ def train(self, df=None, csv_path=None, table_name=None,
             for validation; and `'skip'`, which skips the validation step.
         n_jobs : int, optional
             Number of jobs to use for training and validation. Defaults to
-            4, which allows parallelization over the 4 cross validation folds.
+            `None`, which allows CivisML to dynamically calculate an
+            appropriate number of workers to use.
```

Comment:
I see the specifics are in the full docs, but is it worth mentioning here that "appropriate" generally means "as many as possible without using your entire cluster"?